buildout.cfg

```diff
@@ -1,8 +1,12 @@
 [buildout]
+extensions = mr.developer
 develop = .
 parts = test
-find-links = http://download.zope.org/distribution/
+auto-checkout = *
 
 [test]
 recipe = zc.recipe.testrunner
 eggs = zLOG
+
+[sources]
+zope.testrunner = git https://github.com/zopefoundation/zope.testrunner.git branch=master
```

Comment:
Ugh, is this because of https://github.com/zopefoundation/zope.testrunner/issues/30 ?

I can release zope.testrunner 4.4.10 today so this won't be necessary any more.